File: src/main/java/de/burrotinto/burroPlayer/interfaces/serial/values/ControllBytes.java
package de.burrotinto.burroPlayer.interfaces.serial.values;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * Created by derduke on 14.02.17.
 */
@Setter
@Getter
@Component
@NoArgsConstructor
@AllArgsConstructor
@ConfigurationProperties(prefix="burroplayer.adapter.PI4J.serial.controllBytes")
public class ControllBytes {
    private int startRange;
    private int endRange;
    private int stop;
    private int pause;
    private int status;
    private int random;
}

File: src/main/kotlin/de/burrotinto/burroplayer/interfaces/serial/values/ControllBytes.kt
package de.burrotinto.burroplayer.interfaces.serial.values

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * Created by Florian Klinger on 06.07.17, 21:51.
 */
@Component
@ConfigurationProperties(prefix = "burroplayer.adapter.serial.controllBytes")
data class ControllBytes(var startRange: Int = 0,
                         var endRange: Int = 10,
                         var stop: Int = 128,
                         var pause: Int = 130,
                         var status: Int = 129,
                         var random: Int = 131)
